src/stores/FreshNewStore.ts

```diff
@@ -23,6 +30,95 @@ const useFreshDataStore = create<FreshDataStore>()((set) => ({
     loadConfig: async (configName: string) => {
       set({ config: configName });
     },
+    appFetchesConfigOptions: async () => {
+      try {
+        const availableConfigs = await fetchAvailableConfigs();
+        set({ configOptions: availableConfigs });
+      } catch (error) {
+        // TODO: show a notif error
+        set({ configOptions: [] });
+      }
+    },
+    userSelectsConfig: async (userSelectedConfig: string | null) => {
+      if (userSelectedConfig) {
+        set({ isConfigLoading: true });
+
+        try {
+          const { config, termsData } = await fetchConfig(userSelectedConfig);
+          const namespacePrefix = config.namespace_prefix;
+
+          // Convert standardized variables
+          const standardizedVariables: StandardizedVariables = {};
+          config.standardized_variables.forEach((variable) => {
```

Comment:
I find this use of `.forEach` as a for loop confusing. When I see something like `forEach` or `map`, I expect that you return something to me from the iteration. But here you are using this only for side effects, i.e. to manipulate `standardizedVariables` that is defined outside of the iteration. 

I'd say let's either: 1) use a regular for loop and manipulate `standardizedVariables` inside, or 2) use `.map()` and return `standardizedVariables` from the iteration directly